core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -206,6 +210,10 @@ private FileSystemContext(ClientContext ctx) {
     LOG.info("Created filesystem context with id {}. This ID will be used for identifying info "
             + "from the client, such as metrics. It can be set manually through the {} property",
         mAppId, PropertyKey.Name.USER_APP_ID);
+
+    mWorkerGroup = NettyUtils.createEventLoop(NettyUtils.getUserChannel(ctx.getConf()),
+        ctx.getConf().getInt(PropertyKey.USER_NETWORK_NETTY_WORKER_THREADS),
+        String.format("netty-client-worker-%s-%%d", mAppId), true);
```

Comment:
NIT: would `netty-client-pool` be more meaningful? I'm not sure if we need the word "worker" in there, because is a bit confusing. What do you think?

Reply:
Considering there aren't any other `netty-client-worker-*` threads to be confused with I don't think it makes much of a difference to rename to `netty-client-pool-*`.

If we want to rename this to make it more useful in debugging then I think `alluxio-client-nettyPool-{{appId}-%d` would provide a signficantly more information. It tells us it's a part of the alluxio client, it's part of a nettyPool with the given app ID and the thread number in the pool.

Reply:
hrmmm, can we rename it to that then? That looks good to me.